doc/releasenotes.html

```diff
@@ -59,6 +59,151 @@ <h2>CATEGORY: General, Solar, Detailed Photovoltaic, Wind Power, CSP Physical Tr
 
 -->
 
+<h1 id="2023-12-xx">SAM 2023.12.xx, SSC xxx: December xx, 2023</h1>
+
+<p>This version introduces a set of new hybrid power system models and a new molten salt linear Fresnel models for electric power generation and industrial process heat (IPH) applications.</p>
+
+<p>For a complete list of code revisions associated with this version, see the <a href="https://github.com/NREL/SAM/pulls?q=is%3Apr+milestone%3A%22SAM+Fall+2023+Release%22+is%3Aclosed">SAM</a> and <a href="https://github.com/NREL/ssc/pulls?q=is%3Apr+milestone%3A%22SAM+Fall+2023+Release%22+is%3Aclosed">SSC</a> GitHub repositories.  Highlights include:</p>
+
+<ul>
+<li>New SAM logo and icon.</li>
+<li>New hybrid power models with different combinations of the photovoltaic (PVWatts and Detailed PV), Wind Power, Fuel Cell, and Generic System models with battery storage for the Single Owner and Third Party Host / Developer financial models.</li>
+<li>New concentrating solar power (CSP) and industrial process heat (IPH) model for molten salt linear Fresnel systems.</li>
```

Comment:
I'd consider adding the self-consumption dispatch option as a key bullet point here.